models.py

```diff
@@ -0,0 +1,10 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String
+
+Base = declarative_base()
+
+class FlathubUser(Base):
+    __tablename__ = "users"
+
+    id = Column(Integer, primary_key=True, index=True)
```

Comment:
Did you consider `BigInteger` or a `Guid`?

Reply:
We should use whatever is recommended in SQLAlchemy for this kind of thing.  I'm not sure entirely what is recommended these days.  Guids are not a totally awful idea, but frankly I doubt we'll ever hit enough users to overflow Integer and warrant BigInteger.  Guids would at least offer a bit less predictability in case there were ever a reason to worry about leaking these IDs.